### DISCUSSION TITLE
Apps use non standard colors for Material You theme

### DISCUSSION THREAD

**Discussion by SnoutBug** [FossifyOrg/General-Discussion#223] [General]
All of the apps use colors that differ from standard material you colors. I expect "calendar" or "gallery" to look more similar to every other app that supports m3.

| ![Screenshot_20240718-191037](https://github.com/user-attachments/assets/8da5f48b-7a1b-4562-ae18-efe42eaf853d) | ![Screenshot_20240718-191029](https://github.com/user-attachments/assets/254261c8-5836-4f49-a84e-1cf1356866b6) |
|-------|-------|

As you can see in the screenshots provided the apps look different.
- background color
- dialog color
- etc.

**Comment by Spp595** [comment]
I would like the applications to adapt to the device's material ui theme

**Comment by ghost** [comment]
yes

fossify apps look weirdly inconsistent compared to any other on android, it's kind of jarring

**Comment by naveensingh** [comment]
This will be fixed. Could someone file a [bug report](https://github.com/FossifyOrg/General-Discussion/issues/new?template=bug_report.yml) for it?

**Reply by SnoutBug** [reply]
I opened issue #359. I'm happy that this is getting reworked now, thanks! :)
